public/css/styles.css

```diff
@@ -892,6 +901,29 @@ td, tr, img {
   width: 80%;
 }
 
+.searchbox-top {
+  float: right;
+  border-left: 10px solid #1a1a1a;
+}
+
+.searchbox-top .searchboxinput{
+  border: none;
+  padding: 10px;
+  background-color: #111111;
+  color: #FFF;
+  height: 42px;
+  vertical-align: top;
+}
```

Comment:
I expanded the input (adding more padding to it) to be easier to select by the user. That white outline on focus is default browser way to indicate, what element user has focused. It has strong color because of accessibility (it is especially helpful when you are navigating with keyboard and/or have visual impairement). If needed, it can be changed with "outline" CSS rule (it works similar to "border") on ":focus".
Magnifying glass as button is another focusable element, so I'm not sure if that's good idea to embrace it along (it would also require modifying the form, so the button would be over the text field).